modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/SleepFragment.kt

```diff
@@ -114,6 +112,51 @@ class SleepFragment : BaseDialogFragment() {
             }
         }
 
+        binding.customStepperComposeView.setContentWithViewCompositionStrategy {
+            AppTheme(theme.activeTheme) {
```

Comment:
```suggestion
            AppTheme(Theme.ThemeType.DARK) {
```

The player is always dark so if the active theme is used it might have the wrong colours such as for the press state.


https://github.com/user-attachments/assets/26f95043-0a2f-4c75-88a9-be0070a65983

It's not a big deal, but the pressed state is being clipped, and it might be able to be fixed by attaching to something different in the constraint layout.